### PR TITLE
Account model

### PR DESCRIPTION
**Pull request by Itav8**
Create Account Model

### CODE REVIEW

File: backend/models/guests.py [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
+from sqlalchemy import Column, ForeignKey, Integer, String, Boolean
 from db import Base
 from enum import Enum as PyEnum
 
 
 class Status(PyEnum):
     pending = "pending"
     attending = "attending"
-    decline = "decline"
+    declined = "decline"
```
Comment (misodope) [inline backend/models/guests.py:9]: should the string also be "declined" here?

File: backend/routers/wedding.py [hunk 2]
```diff
@@ -13,7 +12,8 @@
 @router.post("/create/wedding", response_model=WeddingOut | HttpError)
 async def create_wedding(wedding: WeddingCreate, db: Session = Depends(get_db)):
     try:
-        new_wedding = WeddingModel(**wedding.dict())
+        print("INSIDE CREATE", wedding)
```
Comment (misodope) [inline backend/routers/wedding.py:15]: Extra print statement

File: backend/models/guests.py (hunk 1 above)
Comment (Itav8) [reply backend/models/guests.py]: done

File: backend/routers/wedding.py (hunk 2 above)
Comment (Itav8) [reply backend/routers/wedding.py]: done

File: backend/models/guests.py [hunk 3]
```diff
@@ -17,18 +16,17 @@ class GuestModel(Base):
...
     status = Column(String, default=Status.pending.value)
     bride_guest = Column(Boolean)
     groom_guest = Column(Boolean)
     bridesmaids_guest = Column(Boolean)
     groomsmen_guest = Column(Boolean)
     wedding_id = Column(Integer)
     event_type = Column(String)
-
-    # wedding = relationship("WeddingModel", back_populates="guests")
+    # uid = Column(String, ForeignKey("accounts.uid"))
```
Comment (misodope) [inline backend/models/guests.py:32]: Should this be commented back in? `account_uid`

File: backend/models/location.py [hunk 4]
```diff
@@ -22,3 +21,4 @@ class LocationModel(Base):
     location_created = Column(DateTime, server_default=func.now())
     wedding_id = Column(Integer)
     event_id = Column(Integer)
+    # uid = Column(String, ForeignKey("accounts.uid"))
```
Comment (misodope) [inline backend/models/location.py:24]: Same here

**Review by misodope — APPROVED** [review]
Thanks for the updates, LGTM

**Pull request merged**